src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -52,6 +116,33 @@ public void postCreateExternalBinaryContentCopy(final String uri) {
                         "request entity, as indicated by a link with " +
                         "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "copy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.